docs/commands.md

```diff
@@ -21,6 +21,8 @@ Optional args:
    Adding this flag will keep data in an uncompressed form. Compression not only saves
    memory but usually improve performance due to lower number of memory accesses. 
  * CHUNK_SIZE - amount of memory, in bytes, allocated for data. Default: 4000.
+ * DUPLICATE_POLICY - configure what to do on duplicate sample, possible values: BLOCK, FIRST, LAST, MIN, MAX.
```

Comment:
link to #DUPLICATE_POLICY

Reply:
not sure if the options here are necessary if we link it and.  same for the server-wide default, it's explained at the sub section